safe_transaction_service/history/tasks.py

```diff
@@ -472,3 +473,27 @@ def send_webhook_task(address: Optional[str], payload: Dict[str, Any]) -> int:
 
         sent_requests += 1
     return sent_requests
+
+
+@app.shared_task(
+    bind=True,
+    soft_time_limit=SOFT_TIMEOUT,
+    autoretry_for=(MetadataRetrievalExceptionTimeout,),
+    time_limit=LOCK_TIMEOUT,
+    retry_backoff=5,
+    retry_kwargs={"max_retries": 5},
+)
+def chance_to_retrieve_metadata_from_uri(self, uri: str) -> bool:
+    """
+    Give more chances to get the metadata of collectible
+    """
+    from django.core.cache import cache as django_cache
```

Comment:
This can be imported outside the function. The only reason to import stuff inside a function is to prevent recursive imports